src/text_metadata.rs

```diff
@@ -424,7 +414,9 @@ impl ITXtChunk {
             return Err(TextDecodingError::InvalidCompressionMethod);
         }
 
-        let language_tag = decode_ascii(language_tag_slice)?.to_owned();
+        let language_tag = std::str::from_utf8(language_tag_slice)
+            .map_err(|_| TextDecodingError::Unrepresentable)?
+            .to_string();
```

Comment:
What if we _want_ this error on non-ascii-utf8? Afterall, such an image will _not_ decode the same with all other decoders and this is just a guess with regards to the text encoding. There are several other ascii-supersets that may be valid guesses here.